ibc_public/utils_data.py

```diff
@@ -315,7 +315,10 @@ def data_parser(derivatives=DERIVATIVES, conditions=CONDITIONS,
 
     # fixed-effects activation images (postprocessed)
     con_df = conditions
-    contrast_name = con_df.condition
+    try:
+        contrast_name = con_df.condition
+    except AttributeError:
```

Comment:
I don't understand the reason for that. You ssem to want to prevent the function to through an error when con_df.condition is not found. I think that this is bad, because users won't know exactly what dataframe is used.

Reply:
I see. But then we should issue a warning to let the user know.